Diffusion/Common/Pipeline/PipelineLoader.swift

```diff
@@ -109,13 +114,31 @@ extension PipelineLoader {
         }
     }
     
-    // TODO: maybe receive Progress to add another progress as child
+    // TODO: maybe receive Progress to add another progress as child --pcuenca
     func prepare() async throws -> Pipeline {
+//        // Check that the constructed locations match the expected locations
+//        print("url: \(url.path)")
+//        print("filename \(filename)")
+//        print("downloadedURL \(downloadedURL.path)")
+//        print("uncompressURL \(uncompressURL.path)")
+//        print("packagesFilename \(packagesFilename)")
+//        print("compiledURL \(compiledURL.path)")
+//        print("downloaded \(downloaded)")
+//        print("ready \(ready)")
         do {
-            try PipelineLoader.models.mkdir(.p)
+            // Make default models folder if missing
+            let fileExists = FileManager.default.fileExists(atPath: PipelineLoader.models.path)
+            if !fileExists {
+                do {
+                    try FileManager.default.createDirectory(atPath: PipelineLoader.models.path, withIntermediateDirectories: true, attributes: nil)
+                } catch {
+                    print("Error creating PipelineLoader.models path: \(error)")
+                }
+            }
+
```

Comment:
```suggestion
            do {
                try FileManager.default.createDirectory(atPath: PipelineLoader.models.path, withIntermediateDirectories: true, attributes: nil)
            } catch {
                print("Error creating PipelineLoader.models path: \(error)")
            }

```

It's not necessary to check for existence